target/linux/rtl838x/dts/rtl8380_zyxel_gs1900-10hp.dts

```diff
@@ -0,0 +1,224 @@
+/dts-v1/;
+
+#include "rtl838x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "zyxel,gs1900", "realtek,rtl838x-soc";
+	model = "Zyxel GS1900-10HP Switch";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200 mem=128M";
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 67 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_sys: sys {
+			label = "gs1900:green:sys";
+			gpios = <&gpio0 47 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&gpio0 {
+	indirect-access-bus-id = <0>;
+};
+
+&spi0 {
+	status = "okay";
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
```

Comment:
Typically, devices can go faster than 10 MHz and it's a good idea to do so, as this will improve read speed considerably.

Reply:
There is not flash driver support for anything else at the moment.
I would prefer to keep this as is for the moment until more advanced drivers support is implemented.

Reply:
Okay. So far most people just copy-pasted this value, but if you have an actual reason for it, then keep it.